src/main/java/erlyberly/DbgView.java

```diff
@@ -125,6 +139,12 @@ public void initialize(URL url, ResourceBundle r) {
 		
 		seqTraceMenuItem = new MenuItem("Seq Trace (experimental)");
 		seqTraceMenuItem.setOnAction(this::onSeqTrace);
+
```

Comment:
Can you add a JavaFX separator here between the tracing and source viewing items?